src/io/file_descriptor.cr

```diff
@@ -18,6 +18,28 @@ class IO::FileDescriptor < IO
     end
   end
 
+  # :nodoc:
+  def self.from_stdio(fd)
+    # XXX: This is -supposed- to work, but something (libevent?) is changing the FD before here.
```

Comment:
Is this comment still relevant? Is this PR still WIP?

Reply:
The code should still be there, and the comment still stands until someone (me?) looks into the epoll setup that happens for all handles once the first one is initialized. But I'd say the PR is ready